src/components/filterBar/FilterBar.js

```diff
@@ -35,7 +35,7 @@ const FilterBar = (props) => {
       [e.target.name]: value,
     });
   };
-  console.log(filters);
+  // console.log(filters);
 
   return (
     <FilterContainer>
```

Comment:
I think the whole `FilterBar.js` and its folder have been deleted by Winny.. so you can discard this file and all the related imports.

She said she couldn't resolve the issue even with the help from Iryna and the others